README.md

```diff
@@ -15,6 +15,7 @@ The plugin is compatible with the following operating systems:
 - macOS 10.14 or later if you use UE4
 - macOS 12.5 or later if you use UE5
 - Ubuntu 20.04 or later
+- Android ??? or later
```

Comment:
What are the Android platform requirements?

Reply:
Most of the platform requirements do indeed come from Unreal. The only thing that C++SDK restricts further is the requirement for Windows 10 on UE4 (UE4 supports Windows 7). I'm just wondering, looking at https://docs.unrealengine.com/5.2/en-US/android-development-requirements-for-unreal-engine/ and https://docs.unrealengine.com/4.27/en-US/SharingAndReleasing/Mobile/Android/DeviceCompatibility/ , will there be any device that can run UE but cannot run the C++SDK?

Reply:
it would probably the other way around, having devices running c++sdk but not UE. But we are going to bump up the minimum version of Android to 23

Reply:
This will likely mean that Android will not support UE4 which is stuck on 21 and does not build with 23, at least out of the box. Not sure if there is a market for this plugin to work on UE4 though.